### ISSUE TITLE
malignant_compartment_analysis.R

### ISSUE THREAD

**Issue by atharva-bhagwat** [TsankovLab/PM_scRNA_atlas#4]
Cannot install hdWGCNA

https://github.com/smorabit/hdWGCNA/issues/239

**Comment by atharva-bhagwat** [comment]
Installed from source.

**Comment by atharva-bhagwat** [comment]
```bash
Error in `collect()`:
! Failed to collect lazy table.
Caused by error in `db_collect()`:
! Arguments in `...` must be used.
✖ Problematic argument:
• ..1 = Inf
ℹ Did you misspell an argument name?
```

https://github.com/TsankovLab/PM_scRNA_atlas/blob/2e8c9f636cb56a8551b26ecef664f51b03a3535e/scripts/malignant_compartment_analysis.R#L401

**Comment by atharva-bhagwat** [comment]
Fixed after downgrading `dbplyr` to 2.3.4

# ALL DONE
